docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-pg-v4-Y-Z-v4-full-DU.yaml

```diff
@@ -0,0 +1,376 @@
+            isolated: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cpu-isolated" hub}}'
+            reserved: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cpu-reserved" hub}}'
+          hugepages:
+            defaultHugepagesSize: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "hugepages-default" hub}}'
+            pages:
+              - size: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "hugepages-size" hub}}'
+                count: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "hugepages-count" | toInt hub}}'
+          realTimeKernel:
+            enabled: true
+          machineConfigPoolSelector:
+            pools.operator.machineconfiguration.openshift.io/master: ""
+          nodeSelector:
+            node-role.kubernetes.io/master: ''
+    # PTP CONFIG
+    - path: source-crs/PtpOperatorConfig-SetSelector.yaml
+      complianceType: "mustonlyhave"
+      patches:
+      - spec:
+          daemonNodeSelector:
+            node-role.kubernetes.io/worker: ""
+    - path: source-crs/PtpConfigSlave.yaml
+      openapi:
+        path: sno-ran-du/schema.openapi
+      patches:
+        - metadata:
+            name: du-ptp-slave
+          spec:
+            profile:
+            - interface: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "ptpcfgslave-profile-interface" hub}}'
+              name: slave
+              phc2sysOpts: -a -r -n 24
+              ptp4lOpts: -2 -s --summary_interval -4
+            recommend:
+            - match:
+              - nodeLabel: node-role.kubernetes.io/master
+              priority: 4
+              profile: slave
+    # SRIOV OPERATOR CONFIG
+    - path: source-crs/SriovOperatorConfig-SetSelector.yaml
+      complianceType: musthave
+      patches:
+      - spec:
+          configDaemonNodeSelector:
+            node-role.kubernetes.io/master: ""
+          disableDrain: true
+          logLevel: 0
+    # LOCAL STORAGE CONFIG
+    - path: source-crs/StorageLV.yaml
+      patches:
+      - spec:
+          storageClassDevices:
+          - storageClassName: "example-storage-class-1"
+            volumeMode: Filesystem
+            fsType: xfs
+            devicePaths: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "storagelv-devicePaths-1" | toLiteral hub}}'
+    - path: source-crs/StorageClass.yaml
+      patches:
+      - metadata:
+          name: example-storage-class-2
+    # CLUSTER LOGGING
+    - path: source-crs/ClusterLogForwarder.yaml
+      patches:
+      - spec:
+          $patch: replace
+          outputs: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cluster-log-fwd-outputs"  | toLiteral hub}}'
+          pipelines: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cluster-log-fwd-pipelines"  | toLiteral hub}}'
+          filters: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cluster-log-fwd-filters"  | toLiteral hub}}'
```

Comment:
As noted above we should use more fixed content in the source CR (ie be more opinionated) and allow the user to override the URL and labels.